app/services/action_handler.rb

```diff
@@ -0,0 +1,23 @@
+class ActionHandler < BaseService
+  def initialize(payload:, event:, event_type:)
+    @payload = payload
+    @event = event
+    @event_type = event_type
+  end
+
+  def call
+    return unless handleable?
+
+    handle_action
+  end
+
+  private
+
+  def find_user
```

Comment:
I would delete this method. You already have `find_or_create_user`.

Also, this method is only used by the `PullRequest` handler, so in any case, it should go there, right?